### PR TITLE
Temporary fix for getEndpoint exception

### PR DESCRIPTION
**Pull request by marcelveldt**
@MartinHjelmare this fixes the crashing at startup but I don't like it :-)
We can merge this in for now but I'm going to submit another PR to do all the state dumping in smaller pieces (node per node)

### CODE REVIEW

File: src/lib/forward.ts [hunk 1]
```diff
@@ -138,9 +138,17 @@ export class EventForwarder {
           // include metadata and ccVersion in the response
           const metadata = node.getValueMetadata(args);
           args.metadata = metadata;
-          args.ccVersion =
-            node.getEndpoint(args.endpoint).getCCVersion(args.commandClass) ||
-            node.getEndpoint(0).getCCVersion(args.commandClass);
+          try {
+            args.ccVersion =
+              node.getEndpoint(args.endpoint).getCCVersion(args.commandClass) ||
+              node.getEndpoint(0).getCCVersion(args.commandClass);
```
Comment (robertsLando) [inline src/lib/forward.ts:144]:
```suggestion
              node.getEndpoint(args.endpoint)?.getCCVersion(args.commandClass) ||
              node.getEndpoint(0)?.getCCVersion(args.commandClass);
```

In this way you can remove the try/catch

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: src/lib/forward.ts (hunk 1 above)
Comment (marcelveldt) [reply src/lib/forward.ts:144]: isn't endpoint 0 always available as it's the node itself ?

Comment (robertsLando) [reply src/lib/forward.ts]: > isn't endpoint 0 always available as it's the node itself ?

Maybe yes, don't remember, but it doesn't add any overhead so I would keep the `?` :)

**Comment by marcelveldt** [comment]
@robertsLando ready for a final review before merge ?

**Review by robertsLando — APPROVED** [review]
LGTM

**Comment by robertsLando** [comment]
@marcelveldt BTW I would add those infos only when node is ready, but as a temporary fix this is ok :)

**Comment by marcelveldt** [comment]
I've just discussed this on discord with @balloob and we're about to change the logic a bit so we well indeed only request all values info when the node is ready. That will solve this headache issues ;-)

**Comment by robertsLando** [comment]
> That will solve this headache issues ;-)

Good! FYI there is also this: https://github.com/zwave-js/zwavejs2mqtt/issues/347

Actually it's a bug, in poor words it could happen that the manufacturer CC isn't available on node ready event.

**Comment by marcelveldt** [comment]
temporary fix not needed as I'm about to publish a new PR with the definitive fix for this issue.

**Pull request closed without merging**